src/components/common/CityCard.js

```diff
@@ -31,54 +40,70 @@ function CityCard({ city }) {
   };
 
   useEffect(() => {
-    axios
```

Comment:
Is this commented out axios request still needed?

Reply:
just fixed it! thanks!